packages/flutter/lib/src/material/ink_well.dart

```diff
@@ -93,7 +94,8 @@ class InkResponse extends StatefulWidget {
     this.borderRadius: BorderRadius.zero,
     this.highlightColor,
     this.splashColor,
-  }) : super(key: key);
+    this.enableFeedback: true,
+  }) : assert(enableFeedback != null), super(key: key);
```

Comment:
the doc for the constructor should mention that containedInkWell, borderRadius, and enableFeedback can't be null, and the asserts should include the other two as well.